demos/prefsServerIntegration/schemas/schemas.js

```diff
@@ -0,0 +1,39 @@
+/*
+
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/first-discovery/raw/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery");
```

Comment:
I don't think you need to call fluid.registerNamespace here. The call to fluid.defaults should set that all up.